src/launcher/java/org/truffleruby/launcher/RubyLauncher.java

```diff
@@ -0,0 +1,141 @@
+/*
+ * Copyright (c) 2017, 2017, Oracle and/or its affiliates. All rights reserved.
+ * ORACLE PROPRIETARY/CONFIDENTIAL. Use is subject to license terms.
+ */
```

Comment:
The copyright should be adapted.

Reply:
fixed

Reply:
I meant the copyright text/license is wrong (should be our usual tri-license).